libraries/core-react/src/components/Banner/Banner.tokens.ts

```diff
@@ -15,33 +16,57 @@ const {
       primary__moss_green_100: { rgba: infoColor },
       primary__energy_red_100: { rgba: warningColor },
     },
+    ui: {
+      background__default: { rgba: backgroundColor },
+    },
   },
   shape: {
     circle: { minHeight, minWidth, borderRadius },
   },
 } = tokens
 
-export const banner = {
-  enabled: {
-    typography: {
-      ...messageTypography,
-    },
-    spacings: spacingMedium,
-    icon: {
-      info: {
-        background: infoBackground,
-        color: infoColor,
-      },
-      warning: {
-        background: warningBackground,
-        color: warningColor,
-      },
+export type BannerToken = ComponentToken & {
+  icon: {
+    info: {
+      background: string
+      color: string
+    }
+    warning: {
+      background: string
+      color: string
+    }
+    shape: {
+      minHeight: string
+      minWidth: string
+      borderRadius: string
+    }
+  }
```

Comment:
I found putting `icon` under `entities` to be better for conveying sub-entites that are styled or tokenized. The type for icon should also be `ComponentToken` so its "conformed". 

I have found putting on different colors/variants on root level to be easier to maintain. 

Checkout [tokens](https://github.com/equinor/design-system/blob/c6433df9f32fbb2c88bca0e768eda58940551baf/libraries/core-react/src/components/Progress/Circular/CircularProgress.tokens.ts#L20) for `CircularProgress` on suggestion on how to do both of these

Reply:
Maybe we should consider to add a "variant" to the ComponentToken 🤔  We have several components with variants like `warning`, `info` and `error`.

Reply:
Are they variants tho? 🤪  I started a discussion on this in #801 as i ran into the same questions myself when doing progress indicators @wenche

Reply:
I don't know what they are, 🤪  but def. not states like active, disabled and so on. Just put them under variants for now, should be easy to rename to something else. I think we'll see the whole picture with TextArea, Cards ++

Reply:
They are under both color and variants in other components so its a weird one. I am leaning towards that they are _not_ variants, as its just coloring of the component. Variant makes more sense for use-cases such as button `contained` vs. `ghost_icon` where there is a clear visual difference. 

But yeah. lets not change it from what its today, better to do that in one change/release. Albeit we should not colors to variants in future changes imho.

Reply:
Did change it, I think it works.